addon/components/license-picker/component.js

```diff
@@ -12,6 +13,16 @@ export default Ember.Component.extend({
     showCopyrightHolders: true,
     showCategories: true,
     allowDismiss: false,
+    init() {
+        this._super(...arguments);
+        // Debouncing autosave prevents a request per keystroke, only sending it
+        // when the user is done typing (trailing=true), debounce timer can be tweaked.
+        this.set('debouncedAutosave', _.debounce(() => {
```

Comment:
Confirmed that using lodash debounce drastically decreases the number of times 'save' is called. 

I do not think the 'save' action should be called on page load. In preprints, the 'editLicense' action, which is triggered here, is called as soon as the page loads, even though there has really been no change in state. Because we have analytics on the editLicense action, this editLicense count in Keen/GA will be falsely inflated, being triggered every time someone visits this page.